language-plutus-core/generators/Language/PlutusCore/Generators/Internal/Entity.hs

```diff
@@ -73,12 +73,12 @@ data IterAppValue uni head arg r = IterAppValue
     }
 
 instance ( PrettyBy config (Term TyName Name uni ())
-         , PrettyBy config head, PrettyBy config arg, Pretty r
+         , PrettyBy config head, PrettyBy config arg, PrettyConst r
```

Comment:
That was the missing piece.

Reply:
Ah, thanks!  I tried that but failed to add an instance for `EvaluationResult`.